lib/postmark/message_extensions/mail.rb

```diff
@@ -15,6 +15,22 @@ def tag=(val)
       header['TAG'] = val
     end
 
+    #
+    # @val accepts :html_only, :text_only, :html_and_text, :none
+    #
+    def track_links(val = nil)
+
+      val = val.to_s.split('_').map { |e| e.capitalize }.join unless val.nil?
+      default 'TRACK-LINKS', val
+
+    end
+
+    def track_links=(val)
+
+      header['TRACK-LINKS'] = val.to_s.split('_').map { |e| e.capitalize }.join
```

Comment:
Please use `Postmark::Inflector.to_postmark` here instead of duplicating similar code.